### ISSUE TITLE
How to use weights?

### ISSUE THREAD

**Issue by ChrisDelClea** [wwwjk366/gower#11]
Hello @wwwjk366 , 

thanks for this nice package. 
Could you provide an example how to weight specific features? 
What format is required here? 

Best regards
Chris

**Comment by JellyZhangQU** [comment]
> Hello @wwwjk366 ,
> 
> thanks for this nice package. Could you provide an example how to weight specific features? What format is required here?
> 
> Best regards Chris

Hi, @ChrisDelClea 
I am also interested in this part, may I ask if some progress you have got?
Cheers,
Jelly

**Comment by ChrisDelClea** [comment]
@JellyZhangQU ,

yep i made some. You can pass any np.array() with scalers as you wish. I guess one has to choose them whisley. I for example choose a range between an interval of 1 to 10 but i guess also 0.0 - 1.0 would work as long as its in a given range.
